generator/src/main/java/com/scottlogic/deg/generator/restrictions/RowSpec.java

```diff
@@ -16,8 +16,8 @@
  * Typically created by combining choices over a decision tree.
  */
 public class RowSpec {
-    private final ProfileFields fields;
-    private final Map<Field, FieldSpec> fieldToFieldSpec;
+    private ProfileFields fields;
```

Comment:
Why have these fields been made mutable?

Reply:
This has been removed